app/src/main/res/layout/search_activity.xml

```diff
@@ -18,7 +18,7 @@
 
         <EditText
             android:id="@+id/searchText"
-            android:layout_width="357dp"
+            android:layout_width="0dp"
```

Comment:
What is this change doing?

Reply:
With a fixed width, the search bar text would be hidden by the back button. By using the constraints and keeping the width at 0dp, we ensure that this does not happen, no matter what phone will be displaying the search bar text.

Reply:
nice!